test/JDBC/expected/BABEL_4367.out

```diff
@@ -0,0 +1,47 @@
+CREATE TABLE BABEL_4367_1 (
+    id INT
+)
+GO
+
+DROP TABLE BABEL_4367_1, BABEL_4367_2
```

Comment:
let us also add test case where we have 3 tables and middle one does not exists, first one does not exists (both with and without if exists)